workflow/metrics/metrics.go

```diff
@@ -42,6 +42,8 @@ func (s ServerConfig) SameServerAs(other ServerConfig) bool {
 type metric struct {
 	metric      prometheus.Metric
 	lastUpdated time.Time
+	realtime    bool
```

Comment:
metric.IsRealtime() does this already, I'm not sure we should be storing this.

Reply:
`IsRealtime()` is in `wfv1.Prometheus`, but metric here is `prometheus.Metric` and it does not have this function.

Reply:
Ah, yep.